src/AV/Video.cpp

```diff
@@ -1367,10 +1368,10 @@ int Video::addTimer(int delay)
 
 bool Video::addPacket(PacketQueue *queue, AVPacket *packet)
 {
-    av_dup_packet(packet);
+    AVPacket *dup = av_packet_clone(packet);
```

Comment:
Should't this use av_packet_ref() instead? Since it seems that ffmpeg now use ref count, it doesn't need to be copied.